### PR TITLE
fix build issue on s390x

### PR DESCRIPTION
**Pull request by hchenxa**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
the docker image can not be built on s390x as the module of nginx-mod-http-lua and nginx-mod-http-lua-upstream was not available in repo v3.9.

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/kubernetes/issues/84269

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: test/images/echoserver/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,13 @@ FROM BASEIMAGE
 
 CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 
-RUN apk add --no-cache openssl nginx-mod-http-lua nginx-mod-http-lua-upstream
+RUN if [ $(arch) != "s390x" ]; then \
+       apk add --no-cache openssl nginx-mod-http-lua nginx-mod-http-lua-upstream ;\
+    else \
+       apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.8/main openssl nginx-mod-http-lua nginx-mod-http-lua-upstream ;\
```
[the remark this answers is not in the document]
Comment (hchenxa) [reply test/images/echoserver/Dockerfile]: @mkumatag the package `nginx-mod-http-lua` and `nginx-mod-http-lua-upstream` only exist in v3.8 repo for s390x so far.

**Comment by hchenxa** [comment]
@mkumatag @listx I need to change the s390 base image to use nginx:1.13-alpine as:

1) the nginx:1.13-alpine using v3.8 alpine apk repo and have the nginx lua module
2) the nginx:1.15-apline did not have nginx lua module in default alpine apk repo
3) the nginx lua module in v3.8 alpine apk repo have compatibility issues on nginx:1.15-alpine, after some test, only nginx:1.13-alpine can be used now.

File: test/images/echoserver/Dockerfile [hunk 2]
```diff
@@ -17,6 +17,7 @@ FROM BASEIMAGE
 CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 
 RUN apk add --no-cache openssl nginx-mod-http-lua nginx-mod-http-lua-upstream
+
```
Comment (mkumatag) [inline test/images/echoserver/Dockerfile]: why this extra space?

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by hchenxa** [comment]
close this PR and recreate another https://github.com/kubernetes/kubernetes/pull/84502/files

**Pull request closed without merging**